test/tests/php-fpm-hello-web/run.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+set -eo pipefail
+
+dir="$(dirname "$(readlink -f "$BASH_SOURCE")")"
+
+image="$1"
+
+# Build a client image with cgi-fcgi for testing
+client_image="php-fpm-client-$RANDOM-$RANDOM"
+docker build -q -t "$client_image" - > /dev/null <<EOF
```

Comment:
Since we're namespacing, might as well just use the test name now (`librarytest/php-fpm-hello-web:fcgi` or something like that), especially since the image name doesn't need to be amazingly unique (like a container name needs to somewhat reasonably be).